app/models/config.rb

```diff
@@ -134,15 +144,15 @@ def clean_config_value
       # do nothing if empty
       return if config_key.nil? || options.last.nil?
 
-      cleaner = CLEAN_PRE_VALIDATION[config_key.to_sym]
+      cleaners = CLEAN_PRE_VALIDATION[config_key.to_sym]
 
       # no clean function, return
-      return if cleaner.nil?
+      return if cleaners.blank?
 
       # we need to use `method` because `cleaner` is a symbol... this converts
       # the symbol into a Method object, which we can then `call`...
       # See https://ruby-doc.org/core/Object.html#method-i-method
-      method(cleaner).call
+      cleaners.map { |cleaner| method(cleaner).call }
```

Comment:
Pretty minor thing here - generally when I see `map` a reader (for example, me!) is expecting to see some values manipulated and returned. I think in this case the methods are all doing mutation rather than returning values, right? So we can use `each`, which will signal to the reader we don't really care what the return value is, and the interesting stuff is happening via mutation.

Reply:
oops derp yes